### PR TITLE
Allow 'inline' tinymce config option without breaking

### PR DESCRIPTION
**Pull request by Psykar**

### CODE REVIEW

**Comment by krtcom** [comment]
Hi, is this already published? When I do `npm install react-tinymce-input` this is not working and I don't see it in sources either. When do you think this could find its way into npm? Thank you

**Comment by Psykar** [comment]
Yeah, don't think a release has been done yet.
You can just `npm install HurricaneJames/react-tinymce-input` meanwhile.
Although I've noticed this PR doesn't handle setting initial values in the inline version properly.
It would need to be altered to use dangerouslySetHtml for that, and I haven't found the time yet.

**Comment by HurricaneJames** [comment]
Published as part of v2.0, sorry for the delay.

File: src/TinyMCEInput.js [hunk 1]
```diff
@@ -82,7 +82,8 @@ var TinyMCEInput = React.createClass({
       pollInterval: 1000,
       textareaProps: {},
       otherEventHandlers: {},
-      onChange: function() {}
+      onChange: function() {},
+      component: 'textinput',
```
Comment (alysov) [inline src/TinyMCEInput.js:86]: Was the change from "textarea" to "textinput" intentional?  I'm not sure that is valid, seeing how "textinput" is not a valid HTML tag.

Comment (alysov) [reply src/TinyMCEInput.js:86]: Just verified in my project that 1) This is a breaking change (as in it breaks default functionality of this package), and 2) Setting it back to "textarea" fixes it.

Comment (HurricaneJames) [reply src/TinyMCEInput.js:86]: I tested this in my browser, and it seemed to work. I did not notice the change to textinput. I will take a look this evening and see what's going on.

Comment (HurricaneJames) [reply src/TinyMCEInput.js:86]: I tried this on a both Windows and Mac, with Chrome, FF, and IE. It works everywhere. Which OS/browser are you using? Also, which version of React. I just checked both of my environments, and they are 15.5+

Comment (HurricaneJames) [reply src/TinyMCEInput.js:86]: I did manage see what you were talking about and switched the default in 2.1.0

Comment (alysov) [reply src/TinyMCEInput.js:86]: Thank you!

**Comment by Psykar** [comment]
My bad for breaking things. I don't actually recall the reason for the
change right now :(

**Pull request merged**